
Activating Modules:
  1) gcc/4.8.2

Restoring modules to user's mymodules

Activating Modules:
  1) cuda/7.5   3) gmp/6.0         5) mpc/1.0.3   7) python/2.7.10b
  2) cudnn/5.0  4) java/1.8.0_112  6) mpfr/3.1.3

The following have been reloaded with a version change:
  1) gcc/4.8.2 => gcc/4.9.2


Currently Loaded Modules:
  1) parallel_studio_xe_2015/15.0.2  13) intel-mpi/5.1.2.150
  2) binutils/2.25                   14) intel-tbb-oss/intel64/43_20150424oss
  3) openmpi/intel/1.8.4             15) vim/7.4
  4) default-environment             16) python/2.7.10b
  5) marcc-utils/1.0                 17) java/1.8.0_112
  6) slurm/14.11.03                  18) cuda/7.5
  7) VTK/6.3.0                       19) cudnn/5.0
  8) cmake/3.5.0                     20) mpfr/3.1.3
  9) boost/1.60.0                    21) gmp/6.0
 10) opencv/gcc/3.1.0                22) mpc/1.0.3
 11) fftw/2.1.5                      23) gcc/4.9.2
 12) fftw3/intel/3.3.4



Restoring modules to user's mymodules
Using GPU Device:
3
Using TensorFlow backend.
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcublas.so.7.5 locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcudnn.so.5 locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcufft.so.7.5 locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcurand.so.7.5 locally
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties: 
name: Tesla K80
major: 3 minor: 7 memoryClockRate (GHz) 0.8235
pciBusID 0000:85:00.0
Total memory: 11.17GiB
Free memory: 11.11GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: Tesla K80, pci bus id: 0000:85:00.0)
/home-4/tkim60@jhu.edu/.local/lib/python2.7/site-packages/keras/engine/training.py:1573: UserWarning: Epoch comprised more than `samples_per_epoch` samples, which might affect learning results. Set `samples_per_epoch` correctly to avoid this warning.
  warnings.warn('Epoch comprised more than '
